unit-tests/live/d400/test-depth_ae_mode.py

```diff
@@ -9,7 +9,7 @@
 from rspy import tests_wrapper as tw
 
 ctx = rs.context()
-device = test.find_first_device_or_exit();
+device, ctx = test.find_first_device_or_exit();
```

Comment:
Please try using this when not using both params.
![image](https://github.com/user-attachments/assets/99430ee1-62c4-476f-888b-e425d118f3d7)